estate/models/estate_property.py

```diff
@@ -0,0 +1,41 @@
+from odoo import fields, models
+import datetime
+from dateutil.relativedelta import relativedelta
```

Comment:
not critical, but [odoo code guidelines](https://www.odoo.com/documentation/18.0/contributing/development/coding_guidelines.html#imports) give an order in which the imports must be used.